logic/StateManager.py

```diff
@@ -72,9 +72,12 @@ def apply_stats(self, response):
             self.state[0]['player']['stats'][stat] = self.player_stats.stat_dict[stat]
 
         if self.player_stats.sus_int > 50 or self.player_stats.sad_int > 50:
-            self.game.game_screen.game_over()
+            # self.game.game_screen.game_over()
```

Comment:
why is this commented?

Reply:
its cause im bad at the game and i was trying to debug something lol